survey.html

```diff
@@ -63,61 +63,64 @@ <h1>TolakedCFC Survey Form</h1>
                 <option value="other">Other</option>
             </select>
             </div>
+            
                         <!-- RADIO OPTION -->
             <div class="details">
 
-                <label for="radio">Would you recommend TolakedCFC to a friend?</label>
+                    <label for="radio">Would you recommend TolakedCFC to a friend?</label>
                 
-                <label for="recommend-1">
-                    <input type="radio" name="recommend" id="recommend-1"> Yes</input>
-                </label>
+                <div>
+                    <label for="recommend-1"> <input type="radio" name="recommend" id="recommend-1"> Yes</input></label>
+                </div>
 
-                <label for="recommend-2">
-                    <input type="radio" name="recommend" id="Recommend-2"> No </input> 
-                </label>
+                <div>
+                    <label for="recommend-2"> <input type="radio" name="recommend" id="recommend-2"> No </input> </label>
```

Comment:
`<div>
                   <input type="radio" name="recommend" id="recommend-2"/> <label for="recommend-2"> No </label>`